packages/ag-charts-website/src/content/docs/charts-radar-series/index.mdoc

```diff
@@ -63,132 +63,6 @@ axes: [
-    /** yValue as read from series data via the yKey property. */
-    readonly radiusValue?: any;
-    /** yName as specified on series options. */
-    readonly radiusName?: string;
-}
-
-export interface AgRadarSeriesMarkerFormatterParams<DatumType> extends AgSeriesMarkerFormatterParams<DatumType> {
-    angleKey: string;
-    radiusKey: string;
-}
-
-export interface AgRadarSeriesMarkerFormat {
-    fill?: CssColor;
-    stroke?: CssColor;
-    strokeWidth?: PixelSize;
-    size?: PixelSize;
-}
-
-export interface AgRadarSeriesMarker<DatumType> extends AgSeriesMarker {
-    /** Function used to return formatting for individual markers, based on the supplied information. If the current marker is highlighted, the `highlighted` property will be set to `true`; make sure to check this if you want to differentiate between the highlighted and un-highlighted states. */
-    formatter?: AgRadarSeriesMarkerFormatter<DatumType>;
-}
-
-export type AgRadarSeriesMarkerFormatter<DatumType> = (
-    params: AgRadarSeriesMarkerFormatterParams<DatumType>
-) => AgRadarSeriesMarkerFormat | undefined;
-
-export interface AgRadarSeriesLabelFormatterParams {
-    /** The ID of the series. */
-    readonly seriesId: string;
-    /** The value of radiusKey as specified on series options. */
-    readonly value: number;
-}
-
-export interface AgRadarSeriesLabelOptions extends AgChartLabelOptions {
-    /** Function used to turn 'yKey' values into text to be displayed by a label. By default the values are simply stringified. */
-    formatter?: (params: AgRadarSeriesLabelFormatterParams) => string;
-}
-
-export interface AgRadarSeriesTooltip extends AgSeriesTooltip {
-    /** Function used to create the content for tooltips. */
-    renderer?: (params: AgRadarSeriesTooltipRendererParams) => string | AgTooltipRendererResult;
-    format?: string;
-}
-
-export interface AgAngleCategoryAxisOptions extends AgBaseAxisOptions {
-    type: 'angle-category';
-    /** Configuration for the axis ticks. */
-    tick?: AgAxisCategoryTickOptions;
-    /** Shape of grid lines. Default: `polygon` */
-    shape?: 'polygon' | 'circle';
-}
-
-export interface AgRadiusNumberAxisOptions extends AgBaseAxisOptions {
-    type: 'radius-number';
-    /** If 'true', the range will be rounded up to ensure nice equal spacing between the ticks. */
-    nice?: boolean;
-    /** User override for the automatically determined min value (based on series data). */
-    min?: number;
-    /** User override for the automatically determined max value (based on series data). */
-    max?: number;
-    /** Configuration for the axis ticks. */
-    tick?: AgAxisNumberTickOptions;
-    /** Shape of grid lines. Default: `polygon` */
-    shape?: 'polygon' | 'circle';
-    /** Configuration for the title shown next to the axis. */
-    title?: AgAxisCaptionOptions;
-}
-```
+## API Reference
+
+{% interfaceDocumentation interfaceName="AgRadarAreaSeriesOptions" config="{ \"showSnippets\": false }" /%}
```

Comment:
Nit: This should be for `AgRadarLineSeriesOptions` too (there are two series types in this page at the moment)

Reply:
Updated